components/ui/data-table/data-table-expanded-row-content.tsx

```diff
@@ -0,0 +1,57 @@
+import { flexRender } from "@tanstack/react-table";
+import {
+  TableCell,
+  TableRow,
+} from "@/components/ui/table";
+import { Row } from "@tanstack/react-table";
+
+interface DataTableExpandedRowContentProps<TData extends Record<string, unknown>> {
+  row: Row<Record<string, unknown>>;
+  data: TData[];
+  visualization: (props: {className: string, data: TData[], row: Row<Record<string, unknown>>}) => React.ReactNode;
+}
+
+export const DataTableExpandedRowContent = <TData extends Record<string, unknown>>(props: DataTableExpandedRowContentProps<TData>) => {
+  return (
+    <TableRow>
+      <TableCell colSpan={props.row.getVisibleCells().length} className="p-0">
+        <div className="
+          flex
+          w-auto
+          overflow-x-hidden
+          max-w-[95vw]
+          lg:max-w-[79vw]
```

Comment:
Yeah not a very fun magic number. It's because the parent's width is so high I can't possibly use it and I need some kind of measurement that vaguely represents the width of the table which is visible in order to make the stickiness work so that horizontal scrolling always keeps my stuff visible. Not ideal but is the best direction I could find here.

![Peek 2024-05-29 14-21](https://github.com/serotracker/dashboard/assets/86806388/bf2981d9-14f7-479f-9c4f-0c08e7384554)